### ISSUE TITLE
SA1629 (DocumentationTextMustEndWithAPeriod) not recognized on enum fields

### ISSUE THREAD

**Issue by schlotter** [DotNetAnalyzers/StyleCopAnalyzers#3223]
Using StyleCop.Analyzers 1.1.118, the documentaton of `EnumField` does not generate warning [SA1629 (DocumentationTextMustEndWithAPeriod)](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1629.md), although it should.

``` csharp
public enum MyEnum
{
    /// <summary>
    /// Does not generate SA1629 although it does not end in period
    /// </summary>
    EnumField
}
```

**Comment by sharwell** [comment]
Duplicate of #2950
